lib/basic-auth-request.js

```diff
@@ -10,7 +10,8 @@ const buildError = (body) => {
 
 async function getTokenFromBasicAuth (settings) {
   return new Promise((resolve, reject) => {
-    const authUrl = `${settings.url}/oauth/authorize?response_type=token&client_id=openshift-challenging-client`;
+    let authUrl = settings.authUrl ? settings.authUrl : settings.url;
```

Comment:
you can actually do that ternary expression inside the `${}`

like: `${settings.authUrl ? settings.authUrl : settings.url}/......`